pkg/cmd/logs/tail.go

```diff
@@ -0,0 +1,95 @@
+package cmd
```

Comment:
If it's in a subdirectory, it should be its own package. The problem is that you're already using `logs` as a package name.

Not sure what the best solution is but we might want to go with `cmd/logs_tail.go` for now.

cc @brandur-stripe in case there's a more elegant/idiomatic solution you can think of

Reply:
Alternatively, we could rename the `logs` package to something like `logtailing`. Allowing us to name the sub-commands the `logs` package. I don't really like the idea of having sub-commands as top-level commands and think it's a bad pattern to start, even if we have the intention to improve it at some point.